app/src/main/java/ca/marshallasch/veil/FragmentLogin.java

```diff
@@ -45,6 +47,19 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
 
         ConstraintLayout layout = view.findViewById(R.id.login_layout);
 
+        // navigate back to the landing screen
+        view.setFocusableInTouchMode(true);
+        view.requestFocus();
+        view.setOnKeyListener((v, keyCode, event) -> {
+            if (event.getAction() == KeyEvent.ACTION_DOWN) {
```

Comment:
I think you might have to override the default onClick().

Reply:
This works, its the same block of code that you had in the dashboard fragment, but I moved it so that you can go right to the dashboard without having to go through the login screen first